src/core/a-animation.js

```diff
@@ -363,6 +364,8 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
     getForCoordinateComponent();
   } else if (['true', 'false'].indexOf(dataTo) !== -1) {
     getForBoolean();
+  } else if (dataTo && dataTo[0] === '#') {
```

Comment:
We should also use the schema to infer the color type and not just based on the '#' prefix. We could do:
````html
<a-animation attribute="material.color" from="blue" to="red" dur="4000" easing="ease-out"></a-animation>
````